packages/ui/Marquee/MarqueeController.js

```diff
@@ -230,7 +230,6 @@ const MarqueeController = hoc(defaultConfig, (config, Wrapped) => {
 		handleComplete = (component) => {
 			const complete = this.markReady(component);
 			if (complete) {
-				this.cancelJob.stop();
```

Comment:
Seems like this is safe enough.  I think the thought process was that any time we were going to issue a start, we should prevent cancel from stopping it.  But, I think in this instance, it makes sense that if we have a cancel pending, we would want to allow it to prevent the restart.